README.md

```diff
@@ -47,7 +47,7 @@ We provide a helper class to simplify writing inference pipelines using pre-trai
 Here is how we would do it. Run this from the `demo` folder:
 ```python
 from maskrcnn_benchmark.config import cfg
-from predictor import COCODemo
+from demo.predictor import COCODemo
```

Comment:
Ah, I got to know how this works now. I was trying to execute this script in the root directory instead of in the `demo` directory. Since I have installed `maskrcnn_benchmark` as a python package, I shall only care about where I could access to `predictor` with the import rule. I will leave that with its original version.